components/nautobot/values.yaml

```diff
@@ -95,3 +95,47 @@ metrics:
   enabled: true
   prometheusRule:
     enabled: true
+
+extraObjects:
+  - apiVersion: batch/v1
+    kind: CronJob
```

Comment:
I was thinking this would be a job with a hook set so that it's a post-upgrade. @skrobul thoughts?

Reply:
That should work